packages/netsuite-adapter/test/client/suiteapp_client.test.ts

```diff
@@ -591,64 +591,148 @@ describe('SuiteAppClient', () => {
         },
       )).resolves.toBeUndefined()
     })
-  })
 
-  describe('versionFeatures', () => {
-    let client: SuiteAppClient
-    beforeEach(async () => {
-      client = new SuiteAppClient({
-        credentials: {
+    describe('validateFeatures', () => {
+      beforeEach(async () => {
+        mockAxiosAdapter.onPost().reply(200, {
+          status: 'success',
+          results: {
+            errors: [],
+            results: [{ configType: 'FEATURES',
+              data: {
+                fields: {
+                  createsuitebundles: 'T',
+                  tba: 'T',
+                  suiteappdevelopmentframework: 'T',
+                  restwebservices: 'T',
+                  webservicesexternal: 'T',
+                },
+              } },
+            ],
+          },
+        })
+      })
+      it('should succeed if must-have features are enabled', async () => {
+        await expect(SuiteAppClient.validateCredentials({
           accountId: 'ACCOUNT_ID',
           suiteAppTokenId: 'tokenId',
           suiteAppTokenSecret: 'tokenSecret',
           suiteAppActivationKey: 'activationKey',
-        },
-        globalLimiter: new Bottleneck(),
+        })).resolves.toBeUndefined()
       })
-    })
-    it('should set old versionFeatures', async () => {
-      mockAxiosAdapter.onPost().reply(200, {
-        status: 'success',
-        results: {
-          appVersion: [0, 1, 1],
-          time: 1000,
-        },
+
+      it('should fail when a must-have feature is disabled', async () => {
+        mockAxiosAdapter.onPost().reply(200, {
+          status: 'success',
+          result: {
+            results: {
+              configType: 'FEATURES',
+              data: {
+                fields: {
+                  webservicesexternal: 'F',
+                  restwebeservices: 'T',
+                  suiteappdevelopmentframework: 'T',
+                  createsuitebundles: 'T',
+                  tba: 'T',
+                },
+              },
+            },
+          },
+        })
+
+        await expect(SuiteAppClient.validateCredentials({
+          accountId: 'ACCOUNT_ID',
+          suiteAppTokenId: 'tokenId',
+          suiteAppTokenSecret: 'tokenSecret',
+          suiteAppActivationKey: 'activationKey',
+        })).rejects.toThrow()
```

Comment:
expect for a specific message to be thrown
```suggestion
        })).rejects.toThrow('Operation failed because the required webservicesexternal feature is not enabled')
```